utils/zerovec/src/ule/encode.rs

```diff
@@ -113,7 +113,7 @@ unsafe impl<T: VarULE + ?Sized> EncodeAsVarULE<T> for &'_ T {
 
 unsafe impl<T: VarULE + ?Sized> EncodeAsVarULE<T> for Cow<'_, T>
 where
-    T: Clone,
+    T: ToOwned,
```

Comment:
Please also make it `+ ?Sized` so that `Cow<str>` works

Reply:
It's already `<T: VarULE + ?Sized>`

Reply:
ok! It's a bit confusing to have bounds in two different places but you are correct.

Reply:
There actually is a reason I did it like this: `<T: VarULE + ?Sized>` matches all of the other impls, and the `where` is the extra bounds. I didn't jump around for more examples to try to better identify what'd match existing style better.

(In my own code I tend to prefer putting `?Sized` inline and everything else in `where` bounds once things go beyond a single trait bound for a single generic type.)